docs/paypal.md

```diff
@@ -0,0 +1,169 @@
+# Pay with PayPal
+
+Accept PayPal payments in your Android app using the PayPal Payments SDK.
+
+## How it works
+
+This diagram shows how your client, your server, and PayPal interact:
+
+// TODO - Get a diagram of the payment flow similar to [this](https://developer.paypal.com/braintree/docs/start/overview#how-it-works)
+
+## Eligibility
+
+PayPal is available as a payment method to merchants in multiple [countries and currencies](https://developer.paypal.com/docs/checkout/payment-methods/).
+
+This SDK supports a minimum Android API of 23 or higher.
```

Comment:
Did we mean to require min SDK 23 or should this be lowered to 21? The PayPalClient currently requires 23.

Reply:
I also wanted to note I think some work needs to be done to support the latest Android API (31)

Reply:
Unfortunately, the PayPal auth SDK we use for buyer auth has functions that require API 23. The SDK itself still has a minSDK of 21 to allow merchant apps to still build for 21 and 22 but they'll need to do an API check and have a fallback. 

API 31 support is actively being worked on and should be in the next SDK release!